README.md

```diff
@@ -101,7 +101,9 @@ Usually you'll use fastlane by triggering individual lanes:
 Additionally to the `fastlane` commands, you now have access to these `fastlane` tools:
 
 - [`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver): Upload screenshots, metadata, and your app to the App Store
+- [`supply`](https://github.com/fastlane/fastlane/tree/master/supply): Command line tool for updating Android apps and their metadata on the Google Play Store
```

Comment:
Please use the Oxford comma :) http://1.bp.blogspot.com/-2bi5v0sVY4M/Trx92bpZXqI/AAAAAAAAB00/sXs63DJttlE/s1600/grammar%2Bpolice.jpg